classes/OccurrenceListManager.php

```diff
@@ -43,6 +43,7 @@ public function getSpecimenMap($pageRequest, $cntPerPage){
 		if($pageRequest > 0) $pageRequest = ($pageRequest - 1) * $cntPerPage;
 		$sql .= ' LIMIT ' . $pageRequest . ',' . $cntPerPage;
 		//echo '<div style="width: 1200px">' . $sql . '</div>';
+		// echo $sql; exit; // @TODO here
```

Comment:
I'm keeping this in, because I use it pretty much any time the sql for a search breaks, and it always takes me too long to find it again.